Rebase on ansible 2.9

sandrobonazzola opened the pull request:
rebasing the role to ansible 2.9

sandrobonazzola committed:
Rebase on ansible 2.9

rebasing the role to ansible 2.9

sandrobonazzola commented:
@mnecas, @mwperina  can you please review?

sandrobonazzola committed:
automation: use master pipeline only

sandrobonazzola committed:
bump version to 1.1.0

eslutsky reviewed (approved):
looks good

mwperina commented on tasks/main.yml:
```diff
@@ -86,7 +86,7 @@
 
     - name: Get VM list
-      ovirt_vm_facts:
+      ovirt_vm_info:
```
I'm probably missing something here, because you are fetching VMs information, but you are not using this information anywhere. Just be aware that calling ovirt_vm_info doesn't store information about VMs into ansible_facts, so if you really need those information, then you to register a variable to store results of ovirt_vm_info:

https://docs.ansible.com/ansible/latest/modules/ovirt_vm_info_module.html#examples

Otherwise I suggest to remove that invocation

mnecas replied:
Yes exactly, here is another example https://github.com/oVirt/ovirt-ansible-cluster-upgrade/pull/59/files.

tiraboschi replied:
We are fetching the list of VMs to shutdown them, 
see: https://github.com/oVirt/ovirt-ansible-shutdown-env/blob/master/tasks/main.yml#L102

On ansible 2.7/2.8 we were using ovirt_vm_facts that was implicitly registering ovirt_vms fact ( https://docs.ansible.com/ansible/2.7/modules/ovirt_vm_facts_module.html#notes ). ovirt_vm_info doesn't

mwperina commented on tasks/main.yml:
```diff
@@ -103,7 +103,7 @@
 
         - name: Refresh VM list
-          ovirt_vm_facts:
+          ovirt_vm_info:
```
Same comment as above

tiraboschi committed:
Register ovirt_vm_info result

ovirt_vm_facts in ansible 2.8 was implicitly
registering its result on a top-level fact named ovirt_vms:
https://docs.ansible.com/ansible/2.8/modules/ovirt_vm_facts_module.html#notes

with ovirt_vm_info on ansible 2.9 the result has to be explicitely
registered:
https://docs.ansible.com/ansible/latest/modules/ovirt_vm_info_module.html#notes

sandrobonazzola committed:
Merge branch 'master' into master-post-branch
